app/src/main/java/com/elavon/converge/inject/AppModule.java

```diff
@@ -0,0 +1,78 @@
+package com.elavon.converge.inject;
+
+import android.content.Context;
+
+import com.elavon.converge.config.Config;
+import com.elavon.converge.config.ConfigLoader;
+import com.elavon.converge.core.TransactionManager;
+import com.elavon.converge.exception.AppInitException;
+import com.elavon.converge.processor.ConvergeClient;
+import com.elavon.converge.processor.TLSSocketFactory;
+import com.elavon.converge.xml.XmlMapper;
+
+import java.security.KeyStore;
+import java.util.Collections;
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Singleton;
+import javax.net.ssl.TrustManagerFactory;
+import javax.net.ssl.X509TrustManager;
+
+import dagger.Module;
+import dagger.Provides;
+import okhttp3.ConnectionSpec;
+import okhttp3.OkHttpClient;
+import okhttp3.TlsVersion;
+
+@Module
+public class AppModule {
+
+    private final Context context;
+    private final Config config;
+
+    public AppModule(final Context context) {
+        this.context = context;
+        this.config = new ConfigLoader(context.getResources(), Config.Environment.TEST).load();
```

Comment:
environment is harded coded to `TEST` for now. should be picked up from external input.